bus-mapping/src/circuit_input_builder.rs

```diff
@@ -516,42 +574,52 @@ impl<'a> CircuitInputStateRef<'a> {
         });
     }
 
+    /// Reference to the caller Call
+    pub fn caller_call(&self) -> &Call {
+        let (index, _) = self
+            .tx_ctx
+            .call_stack
+            .get(self.tx_ctx.call_stack.len() - 2)
+            .unwrap();
```

Comment:
For getting second last `call_ctx` from `call_stack`, currently it should never fail because we do `handle_call_create` before `gen_associated_ops`.

For other stack values, we do have `?` to propagate the `Result`, so the errors are properly propagated, right?

Reply:
Yes, that makes sense!

The `handle_call_create` was what I missed!